src/App.jsx

```diff
@@ -14,33 +16,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_ALPHABET = 'alphabet';
+const SORT_FIELD_LENGTH = 'length';
```

Comment:
It's not required yet your solution will looks like much better with `enum`
```suggestion
enum SortBy {
  ALPHABET = 'alphabet',
  LENGTH = 'length',
  DEFAULT = ''
}
```